apps/desktop-wallet/src/features/tokenPrices/hooks.tsx

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Should the file be called `tokenPricesHooks.tsx` to help with navigating through the project? 🤔

Reply:
Sure!